### ISSUE TITLE
Enable cssnano and purgecss only for production

### ISSUE THREAD

**Issue by bluesurfer** [tr1s/tris-webpack-boilerplate#22]
It looks like that `purgecss` is active also in development mode. That adds a computational overhead when compiling assets and it gets worse when the CSS size increases.
 
To solve that I change `postcss.config.js` as follows:

```
const purgecss = require('@fullhuman/postcss-purgecss')({
    content: ['./**/*.html'],
    keyframes: true
});

const cssnano = require('cssnano')({
    preset: 'default',
})

module.exports = {
    plugins: [
        require('autoprefixer'),
        ...process.env.NODE_ENV === 'production'
            ? [cssnano, purgecss]
            : []
    ]
};
```
And therefore also change the build command in `package.json`:
```
...

 "build": "NODE_ENV=production webpack -p --config webpack.prod.js --progress --colors",

...

```

it would be nice to have this in the boilerplate :)

**Comment by tr1s** [comment]
Fixed in #25

**Comment by doraemonxxx** [comment]
Hello there. I'm noob and Im using your boilerplate and have a question. 

What does the spread operator do in "...process.env.NODE_ENV === 'production'" ? I don't understand  this kind of checking. When i console.log the "...process.env.NODE_ENV" the value is "p r o d u c t i o n".  
So it is always false and the returned value is "[]" ?? 

Can you pls tell me?
